botbot/apps/plugins/tests/test_logger.py

```diff
@@ -0,0 +1,26 @@
+from django.test import TestCase
+from mock import patch
+from botbot_plugins.base import DummyApp
+
+from botbot.apps.plugins.core import logger
+
+
+class LoggerTestCase(TestCase):
+    def mock_log(**kwargs):
+        args = {}
+        for key in kwargs:
+            args[key] = kwargs[key]
+        return args
+
+    @patch("botbot.apps.plugins.core.logger.Log.objects.create", mock_log)
+    def setUp(self):
+        self.app = DummyApp(test_plugin=logger.Plugin())
+
+    def test_log(self):
+        log = self.app.respond("insert message here")
+        print log
+        self.assertEquals(log['text'], "insert message here", log)
+
+    def test_ignore(self):
+        log = self.app.respond("[off] test message, please ignore")
```

Comment:
Where are you setting the string to be "off" though? I don't even see "off" being given as the default in botbot/apps/plugins/core/logger.py.

Also, could you add a test for /me messages? To make sure it catches e.g. `/me [off] does something off-the-record`.